.github/workflows/test.yaml

```diff
@@ -1194,17 +1199,22 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ vars.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       AWS_ROLE: ${{ vars.AWS_ROLE }}
+      CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
```

Comment:
This can be an env var and does not have to be a secret (as # of secrets are limited)

Reply:
See example above for `AWS_ROLE`

Reply:
Do you agree that we could do the same for the libsql urls, since the auth token is the security factor?

Reply:
Yes! They should look like this `libsql://name.turso.io"` and there is nothing we need to hide here.

Reply:
I will address this in a separate PR right after https://github.com/prisma/team-orm/issues/607